### PR TITLE
[HOLD] Program selection

### PR DESCRIPTION
**Pull request by MrMaksimize**
Ergh.  I"m too tired to write a description.  Lol this is a HOLD till #33 gets merged anyway.  

Still would love some eyes on it.

Next part is to figure out how to send the damn selections and regenerate the page without having to resort to doing the whole thing in backbone.

### CODE REVIEW

**Comment by MrMaksimize** [comment]
Yay! Form is in place now :)

**Comment by MrMaksimize** [comment]
Lol this one just got a lot shorter :)

**Comment by MrMaksimize** [comment]
Gonna keep the hold on this one until it's ready.  Still kinda volatile since I finished the form just now

**Comment by chrisrodz** [comment]
Just let me know when you want reviewing

File: controllers/oppquery.js [hunk 1]
```diff
@@ -22,10 +26,15 @@ var oppQueryCreate = function(req, res, next) {
   });
 };
 
+/**
+ * Get /results
+ * Get the query request, execute, and redner the results page
```
Comment (chrisrodz) [inline controllers/oppquery.js:31]: *render small typo

**Pull request merged**